src/Integration/Persistence/SolutionBindingSerializer.cs

```diff
@@ -97,17 +98,33 @@ private void AddSolutionItemFile(string configFile)
             var projectSystemHelper = this.serviceProvider.GetService<IProjectSystemHelper>();
             projectSystemHelper.AssertLocalServiceIsNotNull();
 
-            Project solutionItemsProject = projectSystemHelper.GetSolutionItemsProject();
+            Project solutionItemsProject = projectSystemHelper.GetSolutionFolderProject(Constants.SonarQubeManagedFolderName, true);
             if (solutionItemsProject == null)
             {
-                Debug.Fail("Could not find the solution items project");
+                Debug.Fail("Could not find the solution items project"); // Should never happen
```

Comment:
I'd remove the comment. `Debug.Fail` already means that it should never happen. :-)